libqtile/backend/base.py

```diff
@@ -10,14 +10,6 @@ class Core(metaclass=ABCMeta):
     def display_name(self) -> str:
         pass
 
-    @abstractmethod
-    def get_keys(self) -> typing.List[str]:
-        pass
-
-    @abstractmethod
-    def get_modifiers(self) -> typing.List[str]:
-        pass
-
```

Comment:
It makes sense for these to be in the base backend, so maybe instead of moving them out of the core into standalone functions, maybe (abstract) static methods would do the job?